Carting/src/WebApi/Controllers/V1/CartController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Carting.WebApi.Application.Carts.Commands.AddItemToCart;
+using Carting.WebApi.Application.Carts.Commands.RemoveItemFromCartCommand;
+using Carting.WebApi.Application.Carts.Queries.GetCart;
+using Carting.WebApi.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Carting.WebApi.Controllers.V1;
+
+[ApiVersion("1")]
+public class CartsController : ApiControllerBase
+{
+    /// <summary>
+    /// Returns a specific cart and its associated items
+    /// </summary>
+    /// <param name="query"></param>
+    /// <returns></returns>
+    /// <remarks>
+    /// Sample request:
+    ///
+    ///     GET /
+    ///     {
+    ///        "CartId": 1
+    ///     }
+    ///
+    /// </remarks>
+    /// <response code="200">Returns the cart</response>
+    /// <response code="404">If the cart does not exist</response>
+    /// <response code="500">Server error</response>
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    [HttpGet]
+    public async Task<ActionResult<CartDto>> GetCart([FromQuery] GetCartQuery query)
+    {
+        return await Mediator.Send(query);
```

Comment:
Fails to meet functional requirements: Version 1 `Returns a cart model (cart key + list of cart items).`

Can't see list of items